crates/bytecode/src/opcode.rs

```diff
@@ -623,9 +623,9 @@ opcodes! {
     // 0xE3
     // 0xE4
     // 0xE5
-    // 0xE6
-    // 0xE7
-    // 0xE8
+    0xE6 => DUPN     => stack_io(0, 1), immediate_size(1);
+    0xE7 => SWAPN    => stack_io(0, 0), immediate_size(1);
+    0xE8 => EXCHANGE => stack_io(0, 0), immediate_size(1);
```

Comment:
The `stack_io` values are not accurate here, they depend on the immediate, unless we want to put the worst case values here. I don't know what this is used for.